### ISSUE TITLE
Bug: Buttons are not having discernible text.

### ISSUE THREAD

**Issue by msftedad** [facebook/react#25949]
React version:
N/A

## Steps To Reproduce
1.Open the Sample app.
2. Open the Accessibility Insights tool for web.
3. Run the fast pass and verify whether the tool is capturing the issue "Ensures buttons have discernible text" or not?

Link to code example:
(https://codesandbox.io/s/react-carousel-a11y-bug-fzov3q)

## The current behavior
Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

## The expected behavior
Ensures buttons have discernible text

![Bug_202670_AI4W](https://user-images.githubusercontent.com/93735775/210058185-1c8b5f7a-b5b2-4639-bef5-71ccf0375080.png)

**Comment by eps1lon** [comment]
You should open this against `react-elastic-carousel` which is released from https://github.com/sag1v/react-elastic-carousel
